machineconfiguration/v1/0000_80_controllerconfig.crd.yaml

```diff
@@ -0,0 +1,1246 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  # name must match the spec fields below, and be in the form: <plural>.<group>
+  name: controllerconfigs.machineconfiguration.openshift.io
+  labels:
+    "openshift.io/operator-managed": ""
+  annotations:
+    api-approved.openshift.io: https://github.com/openshift/api/pull/1453
```

Comment:
So that one wasn't previously in the install manifests, the MCO historically applied it when it came up. 

I've added annotations to match the rest of the CRDs:
```
  include.release.openshift.io/ibm-cloud-managed: "true"
  include.release.openshift.io/self-managed-high-availability: "true"
  include.release.openshift.io/single-node-developer: "true"`
```
I didn't see any harm...well...unless the CVO has some sort of "these are annotated but I didn't apply them" cleanup/pruning function I don't know about.